cmd/transporter/command.go

```diff
@@ -233,17 +233,27 @@ func (c *aboutCommand) Synopsis() string {
 func (c *aboutCommand) Run(args []string) int {
 
 	if len(args) == 0 {
-		for _, a := range adaptor.Adaptors {
-			fmt.Printf("%-20s %s\n", a.Name, a.Description)
+		for name, creator := range adaptor.Adaptors {
+			dummyAdaptor, err := creator(nil, "", map[string]interface{}{"uri": "test", "namespace": "test.test"})
```

Comment:
the map with `uri` and `namespace` is needed as most adaptors check for the presence of them during `init`